README.md

```diff
@@ -1 +1,92 @@
-# pdfgen
\ No newline at end of file
+# pdfgen
+
+An API for creating PDF files both from static HTML and Thymeleaf templates.
+
+### System requirements ###
+
+
+This API requires `wkhtmltopdf` installed on your machine. It should be added to system path. You can download it from [here](https://wkhtmltopdf.org/).
```

Comment:
add info about the specific version requirements (with warining, that some Linux distributions might have older version in their package repositories)